scripts/index.js

```diff
@@ -0,0 +1,109 @@
+
+
+const tempToColor = (temp) => { 
+    if (temp >= 80){
+        return "red-temp";
+    } else if (temp >= 70){
+        return "orange-temp";
+    } else if (temp >= 60){
+        return "yellow-temp";
+    } else if (temp >= 50){
+        return "green-temp";
+    } else {
+        return "teal-temp";
+    }
+    // ((temp>= 70) && (temp <= 79))  
+};
+
+const tempToLandscape = (temp) => {
+    if (temp >= 80){
+        return "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"
+    } else if (temp >= 70){
+        return "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷"
+    } else if (temp >= 60){
+        return "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃"
+    } else {
+        return "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"
+    }
+};
+
+const skyToLandscape = (sky) => {
```

Comment:
I like how you encapsulated this logic in a function that returns the value that you need, and then you can invoke the function and use the return value in the event handler.